lib/install/config/shared.js

```diff
@@ -28,7 +28,7 @@ const config = {
     rules: [
       { test: /\.coffee(\.erb)?$/, loader: 'coffee-loader' },
       {
-        test: /\.js(\.erb)?$/,
+        test: /\.jsx?(\.erb)?$/,
```

Comment:
Agreed, it was just the first file I looked into without completely understanding the process.
The rake task will now do it and this change is not necessary anymore. I will adjust the PR.